src/ReturnTypes/AppMakeHelper.php

```diff
@@ -41,6 +43,11 @@ public function resolveTypeFromCall(FuncCall|MethodCall|StaticCall $call, Scope
                     $resolved = $this->resolve($constantString->getValue());
 
                     if ($resolved === null) {
+                        if (class_exists($constantString->getValue()) || interface_exists($constantString->getValue())) {
+                            $types[] = new ObjectType($constantString->getValue());
+                            continue;
+                        }
```

Comment:
You can directly use the constant string api:

```suggestion
                        if ($constantString->isClassString()) {
                            $types[] = $constantString->getClassStringObjectType();
                            continue;
                        }
```

Reply:
Hm PHPStan didn't like this change

Reply:
Oh you may need `->yes()` on the condition